js/lib/urlutil.js

```diff
@@ -91,30 +91,40 @@ const UrlUtil = {
       return true
     }
 
+    // for cases where we have scheme and we dont want spaces in domain names
+    const caseDomain = /^[\w]{2,5}:\/\/[^\s\/]+\//;
     // for cases, quoted strings
     const case1Reg = /^".*"$/
     // for cases, ?abc and "a? b" which should searching query
     const case2Reg = /^(\?)|(\?.+\s)/
     // for cases, pure string
     const case3Reg = /[\?\.\/\s:]/
-    // for cases, data:uri and view-source:uri
-    const case4Reg = /^\w+:.*/
+    // for cases, data:uri, view-source:uri and about
+    const case4Reg = /^data|view-source|about\:.*/
```

Comment:
I think those might just be examples and not an exhaustive list. We also have registered and app protocols like bitcoin, 1password, etc... @bbondy ?